src/core/modules/NebulaMgr.cpp

```diff
@@ -873,7 +909,7 @@ NebulaP NebulaMgr::searchForCommonName(const QString& name)
 	QString uname = name.toUpper();
 
 	if (const auto it = commonNameMap.find(uname); it != commonNameMap.end())
-		return *it;
+		return *&(it->nebula);
```

Comment:
Why is this `*&` needed? Don't these two operators cancel out?

Reply:
I might need more sleep..., yes.